doc/INSTALL-WINDOWS

```diff
@@ -32,23 +33,24 @@ system.
     Unzip it to your preferred directory
 
 ==== Build
+    - Copy your Jonh the Ripper source code inside C:\cygwin64
```

Comment:
Typo in `Jonh`, and would need to add empty line before this one. Also, this is redundant with "Go to the folder you unzipped John the Ripper source code." that follows - so either don't add this or make more edits to avoid the redundancy.